provider/cmd/manifest.go

```diff
@@ -57,15 +66,75 @@ func doSendManifest(cmd *cobra.Command, sdlpath string) error {
 		return err
 	}
 
-	cert, err := cutils.LoadAndQueryCertificateForAccount(cmd.Context(), cctx, cctx.Keyring)
+	// owner address in FlagFrom has already been validated thus save to just pull its value as string
+	providers, err := providersFromFlagsOrQuery(cmd.Flags(), cmd.Context(), cctx, dtypes.DeploymentID{
+		Owner: cctx.GetFromAddress().String(),
+		DSeq:  dseq,
+	})
+	if err != nil {
+		return err
+	}
+
+	type result struct {
+		Provider sdk.Address `json:"provider" yaml:"provider"`
+		Status   string      `json:"status" yaml:"status"`
+		Error    error       `json:"error,omitempty" yaml:"error,omitempty"`
+	}
+
+	results := make([]result, 0, len(providers))
```

Comment:
can this just do `make([]result, len(providers))` and use `results[i]` in the loop rather than `append` ?

Reply:
I tend to go with the three-argument version to make the loops independent of how it's allocated or if you're actually appending, etc...